add melding handling guidelines

niekijzerman opened the pull request:
- add guidelines to handle meldingen
- add date and time in prompt for agent
- create base path in config file
- fix TYPE ERROR bug (SAI-2352) by type changing prompt template

niekijzerman committed:
feat: add melding handling guidelines

loekadam commented:
Looks good to me, structure is good (thanks for BASEPATH), actual content looks good for now and will probably get some more finetuning along the way.
Do you have an overview of what parts of that big ticket description you do and do not cover with the current content?

niekijzerman commented:
@loekadam cool. I've updated the big ticket description on what this PR covers (and what not).

**In general what is covered in this PR:**
1. policy document now included
2. KeyError 'Type' is fixed

**What still needs to happen:**

1. Make happy flow meldingen
2. Add and finetune additional melding handling guidelines to ensure happy flows work. e.g., (1) how much does a melding need to be 'the same' to be considered duplicate, (2) which bgt_functies are considered non-municipal territory, (3) What policy information is considered relevant and what not
3. Incorporate climate impact framework
4. Make backend English
5. Optimize prompt templates the agent works with (e.g., how much empathy)
6. Improve tool descriptions for central agent
7. Include open-source model switch

I've also added this to the big ticket

loekadam commented:
Nice, this one can be merged I'd say
